scripts/jest/preprocessor.js

```diff
@@ -2,7 +2,9 @@ var assign = require('object-assign');
 var babel = require('babel');
 var babelDefaultOptions = require('../babel/default-options');
 
-var babelOpts = babelDefaultOptions;
+var babelOpts = assign({}, babelDefaultOptions, {
+  retainLines: true,
```

Comment:
Also added this for accurate stack trace line numbers. We do this in **facebook/relay**.